Helper/Webhook.php

```diff
@@ -440,6 +441,8 @@ private function handleOrderTransition(Notification $notification, $transitionSt
     private function handleUnchangedStates(Order $order, Notification $notification): void
     {
         switch ($notification->getEventCode()) {
+            case Notification::REFUND:
+                $this->refundPayment($notification);
```

Comment:
`refundPayment()` is already called in `handleOrderTransition()`. If `handleUnchangedStates()` is called it means that the webhook module has determined that the order is already in the desired state

Reply:
Probably adding closed state to the mapping should be enough to fix the issue